### DISCUSSION TITLE
how I can't prevent users from creating more than 5 posts?

### DISCUSSION THREAD

**Discussion by ooeid** [strapi/strapi#7620] [Legacy Topics]
hello people 

A humble junior developer here who is learning Strapi. 

for starter, I am building a small community gaming website with React. 
I am trying to prevent users from creating more than 5 posts. Any idea how I can do that? 
I can't figure it out if it is React things or Strapi things 

Thank you in advance

**Comment by derrickmehaffy** [comment]
Hi @ooeid in this case you will probably want to look into policies: https://strapi.io/documentation/v3.x/concepts/policies.html

My thinking here is a user logins in, and uses their JWT to make requests from React. Within the policy, you can get the users information from `ctx.state.user` such as the ID and make a query like:

`strapi.query('somemodel').count({ user: ctx.state.user.id })` and check the response for a count of the posts, if >= 5 then do a `return ctx.badRequest('you have exceeded your post count')` else just do an `await next()` to fire the controller and allow them to make a post.
